src/commands/status.rs

```diff
@@ -12,8 +12,8 @@ async fn file_status(
     source: &str,
     snapshots: &std::collections::BTreeMap<String, u128>,
 ) -> fpm::Result<()> {
-    let path = format!("{}/{}", base_path, source);
-    if std::fs::metadata(&path).is_err() {
+    let path = camino::Utf8PathBuf::from(base_path).join(source);
```

Comment:
Change `base_path` type to `camino::Utf8PathBuf` and use `base_path.join(source)`.